integration_test/buckets.tf.template

```diff
@@ -1,5 +1,9 @@
 data "aws_caller_identity" "current_account" {}
 
+provider "aws" {
+  region = "us-west-2"
+}
+
```

Comment:
If we are defaulting to us-west-2, we should also do it in dr-buckets.tf.template.

Also I think default_region can be removed from the bamboo spec files that deploy the buckets.

Reply:
Verify in bamboo-specs/specs/deploy-cumulus-orca-buckets.yaml

Reply:
Added to the dr-buckets.tf.template. Not sure if we want to remove the default_region variable from the bamboo spec files since the scripts used in that reference that variable for ARNs

Reply:
If you have access to bamboo CI, it can be tested without the region to see if it works.